chart/templates/deployment.yaml

```diff
@@ -95,6 +95,10 @@ spec:
         - name: CATTLE_SYSTEM_DEFAULT_REGISTRY
           value: {{ .Values.systemDefaultRegistry }}
 {{- end }}
+{{- if .Values.useBundledSystemChart }}
+        - name: CATTLE_SYSTEM_CATALOG
+          value: bundled
```

Comment:
@leodotcloud The backend environment variable has already been set. This is the PR to enable it through the helm chart installing Rancher. 

I don't think it's worth it to change the env var in the container at this stage of the release. cc: @cjellick @dramich

Reply:
@leodotcloud it is not a boolean, it's explicitly a string value in case we need to add more options in the future. Current options are `bundled` or anything else to use an external resource.

Reply:
@leodotcloud might have been getting at something more like:

```
{{- if .Values.useBundledSystemChart != "false" }}
  - name: CATTLE_SYSTEM_CATALOG
     value: {{ .Values.useBundledSystemChart}}
((- end }}
```
In that case though, I would suggest changing SystemChart = "external" and not set anything for that. Then if bundled or whatever use that.